### PR TITLE
Add Roborock

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

File: entries/r/roborock.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Roborock": {
+    "domain": "roborock.com",
```
Comment (Carlgo11) [inline entries/r/roborock.com.json:3]: https://roborock.com/ doesn't work.
```suggestion
    "domain": "roborock.com",
    "url": "https://www.roborock.com/",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/r/roborock.com.json (hunk 1 above)
Comment (fletcherdodd) [reply entries/r/roborock.com.json:3]: Maybe i'll change it to their global URL (https://global.roborock.com/) instead, the www one goes to their forums 🫤

Comment (Carlgo11) [reply entries/r/roborock.com.json:3]: Really? To me https://www.roborock.com/ redirects to https://us.roborock.com/ :laughing:

Comment (fletcherdodd) [reply entries/r/roborock.com.json:3]: Weird, the 3 times I tried that link it was going to https://forum.roborock.com/ now it is going to the US site 🤷‍♂️

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**